Objects/Movie.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+class Movie: Codable {
+    var id: Int
+    var posterImage: String?
+    var voteAverage: Float
+    var title: String?
+    var releaseDate: String
+    
+    init(posterImage: String, id: Int, voteAverage: Float, title: String?,releaseDate: String) {
```

Comment:
Parameter's value type must be the same as the property's value type. Be attention to the optional!